css/dashboards/dashboards.css

```diff
@@ -43,15 +43,55 @@
     --highcharts-dashboards-dropdown-box-shadow-color: 34, 34, 34, 0.1;
 }
 
+.highcharts-light .highcharts-dashboards-edit-tools .highcharts-dashboards-edit-context-menu-btn {
+    background-color: var(--highcharts-neutral-color-0);
+    filter: none;
+}
+
+.highcharts-light .highcharts-dashboards-edit-tools .highcharts-dashboards-edit-context-menu-btn:hover {
+    background-color: var(--highcharts-neutral-color-20);
+    color: var(--highcharts-neutral-color-0);
+}
+
+/* stylelint-disable max-line-length */
+.highcharts-light .highcharts-dashboards-edit-tools .highcharts-dashboards-edit-button.highcharts-dashboards-edit-tools-btn {
+    background-color: var(--highcharts-neutral-color-0);
+    filter: none;
+    border: 1px solid var(--highcharts-neutral-color-100);
+}
+
+.highcharts-light .highcharts-dashboards-edit-tools .highcharts-dashboards-edit-button.highcharts-dashboards-edit-tools-btn:hover,
+.highcharts-light .highcharts-dashboards-edit-tools .highcharts-dashboards-edit-button.highcharts-dashboards-edit-tools-btn.selected {
+    color: var(--highcharts-neutral-color-100);
+    background-color: var(--highcharts-neutral-color-20);
+    filter: none;
```

Comment:
Do we need to change `color` and `background-color` directly here? CSS variables don't update? I guess it's because it's before their full declaration? The whole class styling should be moved after the 176 line.